js/bitmex.js

```diff
@@ -323,14 +325,11 @@ module.exports = class bitmex extends Exchange {
         }, params);
         let bid = undefined;
         let ask = undefined;
-        if (this.options['fetchTickerQuotes']) {
-            let quotes = await this.publicGetQuoteBucketed (request);
-            let quotesLength = quotes.length;
-            let quote = quotes[quotesLength - 1];
-            bid = this.safeFloat (quote, 'bidPrice');
-            ask = this.safeFloat (quote, 'askPrice');
-        }
-        let tickers = await this.publicGetTradeBucketed (request);
+        let instruments = await this.publicGetInstrument (request1);
```

Comment:
Uhm... we should not really do two requests, unless the user asked for that, so it has to remain optional in case of BitMEX...

Reply:
Anyways, we will still be happy to see more PRs in the future from you! Thx for your involvement!

Reply:
But it *made* previously 2 requests, in previous implementation, before you switched it off in the exchange option since bitmex get quotes/bucketed is completely broken. And now it also make 2 requests but should work. I trying to get freqtrade bot working with bitmex _rzhavym napil'nikom_, it tries to fetch bid and ask values with this function and do not see them there. Current implementation of fetchTicker for bitmex does not return bid and ask values as it shall...

Reply:
BTW, in the reference *official* bitmex client they fetch ticker data via WS api, not REST...